CONTRIBUTING.md

```diff
@@ -195,48 +195,38 @@ installed (`cargo install hyperfine`).
 
 ## Configuring `rust-analyzer`
 
-To configure `rust-analyzer` and VS Code for working on Miri, save the following
-to `.vscode/settings.json` in your local Miri clone:
-
-```json
-{
-    "rust-analyzer.rustc.source": "discover",
-    "rust-analyzer.linkedProjects": [
-        "Cargo.toml",
-        "cargo-miri/Cargo.toml",
-        "miri-script/Cargo.toml",
-    ],
-    "rust-analyzer.check.invocationLocation": "root",
-    "rust-analyzer.check.invocationStrategy": "once",
-    "rust-analyzer.check.overrideCommand": [
-        "env",
-        "MIRI_AUTO_OPS=no",
-        "./miri",
-        "clippy", // make this `check` when working with a locally built rustc
-        "--message-format=json",
-    ],
-    // Contrary to what the name suggests, this also affects proc macros.
-    "rust-analyzer.cargo.buildScripts.invocationLocation": "root",
-    "rust-analyzer.cargo.buildScripts.invocationStrategy": "once",
-    "rust-analyzer.cargo.buildScripts.overrideCommand": [
-        "env",
-        "MIRI_AUTO_OPS=no",
-        "./miri",
-        "check",
-        "--message-format=json",
-    ],
-}
-```
+To configure `rust-analyzer` and the IDE for working on Miri, use one of the provided
+configuration files according to the instructions below.
 
 > #### Note
 >
-> If you are [building Miri with a locally built rustc][], set
+> If you are building Miri with a locally built rustc, set
 > `rust-analyzer.rustcSource` to the relative path from your Miri clone to the
 > root `Cargo.toml` of the locally built rustc. For example, the path might look
-> like `../rust/Cargo.toml`.
+> like `../rust/Cargo.toml`. In addition to that, replace `clippy` by `check`
+> in the `rust-analyzer.check.overrideCommand` setting.
+
+### Visual Studio Code
+
+Copy [`etc/rust_analyzer_settings.json`] to `.vscode/settings.json` in the project root directory.
```

Comment:
What is the point of the brackets around the filename?

These do not become links like this.

Reply:
I thought that it was a style in [rustc-dev-guide](https://rustc-dev-guide.rust-lang.org/building/suggested.html#helix), but it seems that there are just broken links.